wasm/src/lsp/parse.rs

```diff
@@ -1,10 +1,19 @@
-use std::collections::HashMap;
+use std::borrow::Borrow;
+use std::collections::{HashMap, HashSet};
 use std::rc::Rc;
-use clvm_tools_rs::compiler::comptypes::{BodyForm, CompileErr, CompileForm};
-use clvm_tools_rs::compiler::sexp::SExp;
+
+use lsp_types::Position;
+
+use clvm_tools_rs::compiler::compiler::DefaultCompilerOpts;
+use clvm_tools_rs::compiler::comptypes::{BodyForm, CompileErr, CompileForm, HelperForm, LetData, LetFormKind};
+use clvm_tools_rs::compiler::frontend::frontend;
+use clvm_tools_rs::compiler::sexp::{decode_string, parse_sexp, SExp};
```

Comment:
This also should be annotated like this:
```rust
#[cfg(test)]
use clvm_tools_rs::compiler::sexp::{decode_string, parse_sexp};
use clvm_tools_rs::compiler::sexp::{SExp};
```

Reply:
Thanks.